test/e2e/e2e_test.go

```diff
@@ -321,3 +330,90 @@ func recordNames(t *testing.T, records []*resultsv1alpha2.Record) []string {
 	}
 	return ret
 }
+
+func TestGRPCLogging(t *testing.T) {
+	ctx := context.Background()
+
+	// ignore old logs
+	sinceTime := metav1.Now()
+	podLogOptions := corev1.PodLogOptions{
+		SinceTime: &sinceTime,
+	}
+
+	matcher := "\"grpc.method\":\"ListResults\""
+
+	client := newResultsClient(t, allNamespacesReadAccessPath)
+
+	t.Run("Log entry is not found when not expected", func(t *testing.T) {
```

Comment:
I think you meant `Log entry is found when not expected`

Reply:
Indeed. Corrected.

Reply:
LGTM